{"status": "broken", "name": "Merge node with label add label on create", "labels": [{"name": "severity", "value": "normal"}, {"name": "tag", "value": "skip"}, {"name": "feature", "value": "MergeNodeAcceptance"}, {"name": "framework", "value": "behave"}, {"name": "language", "value": "cpython2"}], "stop": 1578486086205, "historyId": "9430954f533827b269dbf08019376847", "start": 1578486085980, "steps": [{"status": "passed", "start": 1578486085982, "stop": 1578486086198, "name": "Given an empty graph"}, {"status": "passed", "start": 1578486086199, "stop": 1578486086202, "name": "When executing query", "attachments": [{"source": "d95db7a9-5f40-408e-b02a-77e5b983e06d-attachment.txt", "type": "text/plain", "name": ".text"}]}, {"status": "broken", "attachments": [{"source": "a959a2f9-d679-4463-96ea-35ab381c50d2-attachment.csv", "type": "text/csv", "name": ".table"}], "stop": 1578486086204, "start": 1578486086202, "statusDetails": {"message": "ResponseError: RedisGraph does not currently support set labels\n", "trace": "  File \"/home/laptop-linux/.local/lib/python2.7/site-packages/behave/model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"/home/laptop-linux/.local/lib/python2.7/site-packages/behave/matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"features/steps/Steps.py\", line 84, in step_impl\n    raise exception\n"}, "name": "Then the result should be"}, {"status": "skipped", "start": 1578486086204, "stop": 1578486086205, "name": "And the side effects should be", "attachments": [{"source": "6d366b41-ef69-423b-a0d6-40facc10ec66-attachment.csv", "type": "text/csv", "name": ".table"}]}], "statusDetails": {"message": "ResponseError: RedisGraph does not currently support set labels\n", "trace": "  File \"/home/laptop-linux/.local/lib/python2.7/site-packages/behave/model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"/home/laptop-linux/.local/lib/python2.7/site-packages/behave/matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"features/steps/Steps.py\", line 84, in step_impl\n    raise exception\n"}, "uuid": "dc999dcb-bfe6-436d-9098-c81889addd8a"}